erpnext/hr/doctype/expense_claim/expense_claim.js

```diff
@@ -332,6 +358,50 @@ frappe.ui.form.on("Expense Claim Advance", {
 	}
 });
 
+frappe.ui.form.on("Expense Taxes and Charges", {
+	account_head: function(frm, cdt, cdn) {
+		var child = locals[cdt][cdn];
+		if(child.account_head && !child.description && !child.rate) {
+			// set description from account head
+			child.description = child.account_head.split(' - ').slice(0, -1).join(' - ');
+
+			// set the tax rate from account head
+			frappe.db.get_value("Account", child.account_head, "tax_rate").then((r) => {
```

Comment:
No need of this code, use fetch from property to fetch rate from the account master

Reply:
added changes